code/modules/mob/living/carbon/human/human_emote.dm

```diff
@@ -648,6 +648,31 @@
 	muzzled_noises = list("annoyed")
 	emote_type = EMOTE_SOUND | EMOTE_MOUTH
 
+/datum/emote/living/carbon/human/thiss
```

Comment:
You should be able to make this a subtype of the existing unathi hiss emote, and just change the different values. This would let the emote key still be `hiss`, rather than `thiss`. See how things are handled below with the `crack` emote.

Reply:
i see! i'm assuming i'd be able to do this if i add "/tajaran" to the datum?